hack/install-deps.sh

```diff
@@ -125,3 +127,9 @@ checkout_repo ${CONTAINERD_PKG} ${CONTAINERD_VERSION}
 cd ${GOPATH}/src/${CONTAINERD_PKG}
 make
 ${sudo} make install -e DESTDIR=${CONTAINERD_DIR}
+
+#Install crictl
+checkout_repo ${CRICTL_PKG} ${CRICTL_VERSION}
+cd ${GOPATH}/src/${CRICTL_PKG}
+make crictl
+${sudo} cp ${GOPATH}/bin/crictl ${CONTAINERD_DIR}/bin/ 
```

Comment:
No need to copy from go/bin to usr/local/bin both should be in the path

Reply:
1) In hack/install-deps.sh we don't check whether `${GOPATH}/bin` is in `PATH`;
2) We also use this to install binaries into `release-stages`

I think that's why we are doing this here. :)

Reply:
hmm..Just checking my path is /home/mike/go/bin:/usr/local/bin for non root login and /usr/local/bin:/home/mike/go/bin for root.

So if I ran this sh one day then ran make on critools/ or go get etc.. then I'd be busted for root, but work ok when non root?

Am I missing something? Is the containerd_dir always in _output?

Reply:
I believe `make crictl` will also install it to your GOPATH, right?
So you'll have the same version in both GOPATH and /usr/local/bin.

Reply:
The first time you build via make there will be no copy.  Then if you ever run install deps there will be a copy, and for a moment in time it's the same copy.  Then some time after making the copy when you build again you now have two different copies and probably won't be thinking about re-running install deps.